marathon-bigip-ctlr.py

```diff
@@ -389,6 +422,7 @@ def __init__(self, appId, servicePort, healthCheck):
         self.iappOptions = {}
         self.mode = 'tcp'
         self.balance = 'round-robin'
+        self.source_addr_translation = {'type': 'automap'}
```

Comment:
If you change this to a json string then the get_source_addr_translation() can be simplified without the special if statement.

Reply:
Sounds good I'll change it.